src/Waives.Http/Document.cs

```diff
@@ -100,5 +149,18 @@ public async Task<ExtractionResults> Extract(string extractorName)
             var responseBody = await response.Content.ReadAsAsync<ExtractionResults>().ConfigureAwait(false);
             return responseBody;
         }
+
+        private static string ResultsFormatToMimeType(ReadResultsFormat format)
```

Comment:
:muscle: This is a good candidate for an extension method, as per https://stackoverflow.com/a/479453